announcements/2021/week05.md

```diff
@@ -0,0 +1,12 @@
+# :zap: This week in provider-independent announcements (January 29, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[External DNS v2.0.0](waiting for link) allows customers to install and use External DNS for their own purpose, separate from the preinstalled app in AWS and Azure clusters. 
+
+[Cert Manager v2.4.2](waiting for link) enables automated deletion of secrets when the parent Certificate is deleted, reducing unnecessary alerts.
```

Comment:
@glitchcrab @giantswarm/app-squad-cert-manager pls check!

Reply:
Link is not available. Asking in https://github.com/giantswarm/giantswarm/issues/15520

Reply:
Yup looks good - maybe worth saying that the deletion is set to `false` by default? Makes it clear that this is opt-in behaviour.

Reply:
It's here: https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.4.2/

Reply:
@glitchcrab @marians thanks! I'm changing it to

> [Cert Manager v2.4.2](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.4.2/) provides the option to automatically delete secrets when the parent Certificate is deleted, reducing unnecessary alerts.